package-lock.json

```diff
@@ -1126,213 +1126,198 @@
+      "version": "1.3.2",
+      "resolved": "https://registry.npmjs.org/@bbc/psammead-locales/-/psammead-locales-1.3.2.tgz",
+      "integrity": "sha512-ZtYRcRs7n/zyJOWlv9hDiAjK9k8KSqoXzlK/iEH541lHYpTM5Oi+dvAOtmH6H7Q+cJyL1JY2VIzSoDoM0HHCwA=="
     },
     "@bbc/psammead-media-indicator": {
-      "version": "2.4.1",
-      "resolved": "https://registry.npmjs.org/@bbc/psammead-media-indicator/-/psammead-media-indicator-2.4.1.tgz",
-      "integrity": "sha512-oGhgjlOORUBjFKat0//kTw1HJc3hSBc3DugWYK0NAN0s8ip32kHwuhhnL7jaO4OX68JHGJ0yNrPyTJsr18+e5w==",
+      "version": "2.5.3",
+      "resolved": "https://registry.npmjs.org/@bbc/psammead-media-indicator/-/psammead-media-indicator-2.5.3.tgz",
+      "integrity": "sha512-ZWCyUUbotyRrCaF0cfxOL9a+lavwbpN0tUnlxkEz0J+tgB2G3KpDwcGaITa9jG/sVwGpEL6xQnCwfGKQafld1Q==",
       "requires": {
-        "@bbc/gel-foundations": "^3.2.1",
-        "@bbc/psammead-styles": "^2.0.2"
+        "@bbc/gel-foundations": "^3.3.2",
+        "@bbc/psammead-styles": "^2.1.1"
       }
     },
     "@bbc/psammead-navigation": {
-      "version": "2.1.4",
-      "resolved": "https://registry.npmjs.org/@bbc/psammead-navigation/-/psammead-navigation-2.1.4.tgz",
-      "integrity": "sha512-EV6D+e/u1vJH0fejOevodzknfqU/HKXBw52pg1KF7KyeQdVXvRpZsMr0pCyABx32D02vaZsA6gDG2tiEhIUUmw==",
+      "version": "2.2.3",
+      "resolved": "https://registry.npmjs.org/@bbc/psammead-navigation/-/psammead-navigation-2.2.3.tgz",
+      "integrity": "sha512-jMho1dzQo8p2dAFgsoldDqI8iNSv5+MJRic9zuM+wDnze3kQF+yOjL6j60i78gcDOTjGhEBhbB+86FQxE+t1qg==",
       "requires": {
-        "@bbc/gel-foundations": "^3.2.1",
-        "@bbc/psammead-styles": "^2.0.2",
-        "@bbc/psammead-visually-hidden-text": "^1.1.2"
+        "@bbc/gel-foundations": "^3.3.2",
+        "@bbc/psammead-styles": "^2.1.1",
+        "@bbc/psammead-visually-hidden-text": "^1.2.1"
       }
     },
     "@bbc/psammead-paragraph": {
-      "version": "2.1.4",
-      "resolved": "https://registry.npmjs.org/@bbc/psammead-paragraph/-/psammead-paragraph-2.1.4.tgz",
-      "integrity": "sha512-UaxHfD/EJnHvQHml/jNaiPShZIvWlZ9y7BpU3Teiq7p8sxco9aDZz5CI8OJGTHs6gr/fQvp4Gb+YMN5jbEFKmQ==",
+      "version": "2.2.3",
+      "resolved": "https://registry.npmjs.org/@bbc/psammead-paragraph/-/psammead-paragraph-2.2.3.tgz",
+      "integrity": "sha512-Aqd5q2PffeNtYcZV0nMBY4nHc3Gq+eRIYGhlzssHr0MfTAwRwMLQhe/HM6VFgEGt2gHv6qZJ2ARfxx5n5zclLA==",
       "requires": {
-        "@bbc/gel-foundations": "^3.2.1",
-        "@bbc/psammead-styles": "^2.0.2"
+        "@bbc/gel-foundations": "^3.3.2",
+        "@bbc/psammead-styles": "^2.1.1"
       }
     },
     "@bbc/psammead-section-label": {
-      "version": "2.1.5",
-      "resolved": "https://registry.npmjs.org/@bbc/psammead-section-label/-/psammead-section-label-2.1.5.tgz",
-      "integrity": "sha512-ffdpDFlraL+eBdYkmneZmyzkusWi06vM7rj2xb3jSgWw2Ntlw+rBQOnHqZAePI84iRbHb9eZ3ll0A9uZ3hT2fA==",
+      "version": "2.3.3",
```

Comment:
you might need to remove the caret from the package-lock; it still seems to have picked >2.1.6 :(